src/Core/DesignTokens/Services/IThemeStorageService.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Microsoft.Fast.Components.FluentUI.DesignTokens;
+
+internal interface IThemeStorageService : IAsyncDisposable
```

Comment:
What's the difference between this IThemeService and IThemeStorageService? 
The persistence service should be outsourced. It is not the purpose of this library to use LocalStorage. The best would be to have a `ThemeSerialized` property (for example) that allows you to retrieve a json with all the theme data, and update this property to reapply the theme.

Reply:
Seems like that's understandable, how about we create an nullable initial-data parameter that FluentThemeProvider component accepts it.

If we want to go an extra step, we can include a model aka Data Blueprint that the users pass directly to FluentThemeProvider component as initial data that we could have instant access to without serialization since the user does the serialization/deserialization from wherever they keep that data persistent.

In this case, if we have initial data we apply the theme based on that or revert back to default

Reply:
I have added Initial Data Support on the component and Model on Latest Commits, feel free to take a look and let me know. If it's ideal on this Scenerio